client/write_state.go

```diff
@@ -94,7 +49,7 @@ type writeState struct {
 
 	session           *session
```

Comment:
As discussed maybe we can remove this now and replace with direct ref to the remaining pool and the consistency write level.